ui/App.jsx

```diff
@@ -0,0 +1,163 @@
+import React, { Suspense, lazy } from 'react';
+import { Route, Switch, useLocation, Redirect } from 'react-router-dom';
+import { hot } from 'react-hot-loader/root';
+import { ConnectedRouter } from 'connected-react-router';
+import { Provider } from 'react-redux';
+import { Helmet } from 'react-helmet';
+
+import { configureStore, history } from './job-view/redux/configureStore';
+import LoadingSpinner from './shared/LoadingSpinner';
+import LoginCallback from './login-callback/LoginCallback';
+import TaskclusterCallback from './taskcluster-auth-callback/TaskclusterCallback';
+import UserGuideApp from './userguide/App';
+
+const IntermittentFailuresApp = lazy(() =>
+  import('./intermittent-failures/App'),
+);
+const PerfherderApp = lazy(() => import('./perfherder/App'));
+
+const PushHealthApp = lazy(() => import('./push-health/App'));
+
+const JobsViewApp = lazy(() => import('./job-view/App'));
+
+const LogviewerApp = lazy(() => import('./logviewer/App'));
```

Comment:
Lazy loading of assets will keep our initial asset entry size about the same as what it was with the multiple entry points.